### ISSUE TITLE
Tech Used Note

### ISSUE THREAD

**Issue by originell** [TEDxVienna/2014-mainsite-wptheme#7]
I know it's not necessary or planned in any way, but maybe a "Powered by" subpage (or small note in the footer) would be nice. It's always hard to get an answer to "how did they do that?" "what might they be using".. making the backend more transparent to interested people. Just a small list "Hey, we are using wordpress and these plugins which do exactly that here"

Or maybe we should just simply put it into the console haha

![image](https://cloud.githubusercontent.com/assets/23529/6199594/7c6a66ca-b44d-11e4-85a4-bf8c3eea52ab.png)

**Comment by zietho** [comment]
Good idea, really like it - what about: https://wordpress.org/plugins/wp-pluginsused/screenshots/

**Comment by IchFloque** [comment]
agree, but let’s do that at the end

**Comment by zietho** [comment]
yeah sure, just looked through all the issues :D
